plan/new.go

```diff
@@ -52,7 +52,23 @@ func buildTestCases(plan *Plan) []TestCase {
 				Plan:      plan,
 				Client:    client,
 				Arguments: testArgs,
+				Skip:      false,
 			}
+
+			for _, filter := range behavior.SkipFilters {
+				// Each filter should be a complete match
+				isCompleteMatch := true
+				for key, value := range filter.AxisMatches {
+					if (key == behavior.ClientAxis && value != client) ||
```

Comment:
Nope. I'm checking all the axis. testArgs does that. testArgs has everything but 0th argument that is client axis.

Well, I'm using the map property in validateAndApplyFilters when I need to get all the skip filters for a particular behavior.
Note the map is behavior -> slice of Filters. With each filter in Filters being a map.